qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -148,6 +148,37 @@ def calibrations(self) -> Calibrations:
         """Return the calibrations."""
         return self._cals
 
+    @property
+    def analysis(self) -> Union[BaseAnalysis, None]:
+        return self._analysis
+
+    @analysis.setter
+    def analysis(self, analysis: Union[BaseAnalysis, None]) -> None:
+        """Set the analysis instance for the experiment"""
```

Comment:
Can you add a bit of doc here to state that this is here so that update calibrations is called and so that this also works with composite experiments.

Reply:
note section is added to the property method rather than here. I think setter method doesn't appear in the API doc and one needs to read the code anyways to find the comment. Note that such comment is already in the method. Now note will appear with the updated property method.